src/Chainweb/Pact/Utils.hs

```diff
@@ -64,15 +63,16 @@ toTxCreationTime (Time timespan) =
   TxCreationTime $ ParsedInteger $ fromIntegral $ timeSpanToSeconds timespan
 
 
-validatePubKey :: P.PublicKeyText -> Bool
-validatePubKey = validateKeyFormat
 
+-- TODO: This will fail for k: accounts that correspond to WebAuthn public
+-- keys. Consider extending it when we integrate WebAuthn with k: accounts.
```

Comment:
Also here: TODO comments are usually just a record of a thought, rarely acted upon. If you really want to make this happen, Asana is the place!

Reply:
https://app.asana.com/0/1204132908184958/1206014030451021/f